File: datas/templates/PHP/Qt Form/$Class Name$.php
<?php
include( 'ui_$Class Name$.php' );

class $Class Name$ extends $Qt Widget Class$
{
    public $signals = array( "" );
    public $slots = array( "" );
    protected $ui;

    public function __construct( QWidget $parent = NULL )
    {
        parent::__construct( $parent );
        $ui = new Ui_$Class Name$();
        $ui->setupUi( $this );
    }
    
    public function __destruct()
    {
        unset( $ui );
        parent::__destruct();
    }
 }
?>

File: datas/templates/C++/Qt Form - Multiple Inheritance/$Class Name$.cpp
#include "$Class Name$.h"

#include <QEvent>

$Class Name$::$Class Name$( QWidget* parent )
    : $Qt Widget Class$( parent )
{
    setupUi( this );
}

$Class Name$::~$Class Name$()
{
}

void $Class Name$::retranslateUi()
{
    Ui_$Class Name$::retranslateUi( this );
    // do your custom retranslate here
}

void $Class Name$::changeEvent( QEvent* event )
{
    $Qt Widget Class$::changeEvent( event );
    
    if ( event->type() == QEvent::LanguageChange ) {
        retranslateUi();
    }
}
